Makefile

```diff
@@ -302,7 +322,13 @@ go-build-linux: export GOARCH=amd64
 go-build-linux: $(BINARIES)
 
 .PHONY: run
-run: docker-build kind-cluster kind-load kind-deploy #HELP Build the operator-controller then deploy it into a new kind cluster.
+run: generate docker-build kind-cluster kind-load kind-deploy wait #HELP Build the operator-controller then deploy it into a new kind cluster.
```

Comment:
I am curious why `generate` was not part of the `run` before?

Reply:
It was for catalogd but not for oper-con
But, I think we should right have that

Reply:
It does actually run already as a dependency, it was hard to find it but:
`docker-build` runs `build-linux`, which runs `build-deps`, which runs `manifests` and `generate`.

Reply:
I removed